test/pages/LandingProduct.test.tsx

```diff
@@ -110,21 +109,4 @@ describe('Get all products', () => {
 		).toBeInTheDocument();
 		expect(screen.getByPlaceholderText('Maximum price')).toBeInTheDocument();
 	});
```

Comment:
Why Have you deleted this test?

Reply:
The test owner Bonheur told me to do so as it was bringing more issues